src/format/reveal/format-reveal.ts

```diff
@@ -37,6 +37,7 @@ import {
   revealMuliplexPreviewFile,
   revealMultiplexExtras,
 } from "./format-reveal-multiplex.ts";
+import { info, warning } from "https://deno.land/std@0.97.0/log/mod.ts";
```

Comment:
this should be imported from `log/mod.ts` (not the full url)

Reply:
This should be removed in fact